spec/workers/cds_updates_synchronizer_worker_spec.rb

```diff
@@ -122,11 +80,11 @@
         context 'when after cut off time' do
           let(:cut_off_time) { 5.minutes.ago }
 
-          it { expect(TariffSynchronizer).to have_received(:download_cds) }
-          it { expect(TariffSynchronizer).to have_received(:apply_cds) }
+          it { expect(CdsSynchronizer).to have_received(:download) }
+          it { expect(CdsSynchronizer).to have_received(:apply) }
 
-          it { expect(TariffSynchronizer).not_to have_received(:download) }
-          it { expect(TariffSynchronizer).not_to have_received(:apply) }
+          it { expect(TaricSynchronizer).not_to have_received(:download) }
```

Comment:
I don't think these tests are relevant anymore since this is the Cds synchronizer

Reply:
It is basically testing taric synchronizer doesn't get called by cds update worker. So I think it is relevent negative test.